node-tests/.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
We should be able to remove this file by adding an overrides config that matches `node-tests/**` with these settings to the top level `.eslintrc.js` file...